jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListViewImpl.java

```diff
@@ -421,8 +441,57 @@ public void onClick( ClickEvent event ) {
 
         controlBulkOperations();
     }
+    
+    private Column<ProcessInstanceSummary, ProcessInstanceSummary> initErrorCountColumn() {
+        
+        Column<ProcessInstanceSummary, ProcessInstanceSummary> column = new Column<ProcessInstanceSummary, ProcessInstanceSummary>(
+                new AbstractCell<ProcessInstanceSummary>(BrowserEvents.CLICK){
+
+            @Override
+            public void onBrowserEvent(Context context, Element elem, ProcessInstanceSummary val, NativeEvent event,
+                    ValueUpdater<ProcessInstanceSummary> valueUpdater) {
+                super.onBrowserEvent(context, elem, val, event, valueUpdater);
+                if(BrowserEvents.CLICK.equals(event.getType()) && val != null) {
+                    errCountPopup.show(val.getProcessInstanceId(), val.getErrorCount(), event.getClientX(), event.getClientY());
```

Comment:
@rbarriuso sorry I missed this on my first pass but perhaps a more suitable component would be to use a popover instead of a popup: http://www.patternfly.org/pattern-library/widgets/#popover